src/cgame/cg_gameinfo.cpp

```diff
@@ -41,11 +41,11 @@ CG_ParseInfos
 int CG_ParseInfos( const char *buf, int max, char *infos[] )
 {
 	char *token;
-	int  count;
+	int  count = 0;
 	char key[ MAX_TOKEN_CHARS ];
 	char info[ MAX_INFO_STRING ];
 
-	count = 0;
+	int infoPostfixLen = strlen( "\\num\\" ) + strlen( va( "%d", MAX_ARENAS ) );
```

Comment:
Ok, replaced some BG_Alloc calls to malloc in low-level functions https://github.com/dimhotepus/Unvanquished/commit/15ec4d5aa0a779563b508a3e2a92709a1beb2555.